src/views/index.hbs

```diff
@@ -6,11 +6,169 @@
 			</h1>
 		</div>
 
+
+    	<!--charts-->
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.4/Chart.js"></script>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/chartjs-plugin-datalabels@0.4.0/dist/chartjs-plugin-datalabels.min.js"></script> 
+    
+
+
+    <!--pie chart-->
+  
+     <div style="margin-left:5%;margin-right:5%">
+        <canvas id="myChart" style="width:100%;max-width:80%;"></canvas>
+    </div>
+
+  <script>
+  var xValues = ["Flask", "React.js", "Unity", "USP.NET Core", "Laravel", "Drupal", "OWL(Odoo Web Library)", "Odoo", "Spring", "jQuery", "Vue.js", "Angular.js", "Angular", "Express", "FastAPI", "Django"];
+  var yValues = [19, 7, 4, 4, 4, 4, 4, 4, 4, 4, 7, 7, 11, 4, 7, 7,];
```

Comment:
I will suggest you use `let` everywhere you have used `var` unless you are dealing with a variable that will not be modified then use `const`.
```suggestion
  let yValues = [19, 7, 4, 4, 4, 4, 4, 4, 4, 4, 7, 7, 11, 4, 7, 7,];
```

Reply:
Noted